tests/test_prices.py

```diff
@@ -139,31 +152,60 @@ def test_compare_taxed_money_field_with_django_field():
 @pytest.mark.parametrize(
     "data,initial,expected_result",
     [
-        ("5", Money(5, "BTC"), False),
-        ("5", Money(10, "BTC"), True),
-        ("5", "5", False),
-        ("5", "10", True),
-        ("5", None, True),
-        (None, Money(5, "BTC"), True),
-        (None, "5", True),
-        (None, None, False),
+        (["5", "BTC"], Money(5, "BTC"), False),
+        (["5", "BTC"], Money(10, "BTC"), True),
+        (["5", "BTC"], ["5", "BTC"], False),
+        (["5", "BTC"], ["5", "USD"], True),
+        (["5", "BTC"], None, True),
+        ([None, None], Money(5, "BTC"), True),
+        ([None, None], ["5", "BTC"], True),
+        ([None, None], None, False),
     ],
 )
 def test_form_changed_data(data, initial, expected_result):
-    form = RequiredPriceForm(data={"price_net": data}, initial={"price_net": initial})
+    form = RequiredPriceForm(
+        data={"price_net_0": data[0], "price_net_1": data[1]},
+        initial={"price_net": initial},
+    )
     assert bool(form.changed_data) == expected_result
 
 
-def test_render():
-    widget = widgets.MoneyInput("BTC", attrs={"type": "number"})
-    result = widget.render("price", 5, attrs={"foo": "bar"})
-    attrs = ['foo="bar"', 'name="price"', 'type="number"', 'value="5"', "BTC"]
+@pytest.mark.parametrize(
+    "field,value,expected_result",
+    [("price_net_0", "5", True), ("price_net_1", "USD", True)],
+)
+def test_form_changed_one_data(field, value, expected_result):
+    form = RequiredPriceForm(
+        data={field: value}, initial={"price_net": Money(10, "BTC")}
+    )
+    assert bool(form.changed_data) == expected_result
+
+
+def test_render_money_input():
+    widget = widgets.MoneyInput(choices=[("USD", "US dollar")], attrs={"key": "value"})
+    result = widget.render("price", Money(5, "USD"), attrs={"foo": "bar"})
+    attrs = [
+        'foo="bar"',
+        'name="price_0"',
+        'name="price_1"',
+        'key="value"',
+        'value="5"',
+        "USD",
+    ]
+    for attr in attrs:
```

Comment:
This test would be six times as readable if it was six separate assertion statements.